relay/kusama/src/lib.rs

```diff
@@ -1415,6 +1429,7 @@ impl InstanceFilter<RuntimeCall> for TransparentProxyType {
 			(x, y) if x == y => true,
 			(ProxyType::Any, _) => true,
 			(_, ProxyType::Any) => false,
+			(ProxyType::Staking, ProxyType::StakingOperator) => true,
```

Comment:
Shouldn't it be reverse? `StakingOperator` is superset of Staking, right? 

https://paritytech.github.io/polkadot-sdk/master/frame_support/traits/trait.InstanceFilter.html#method.is_superset